examples/hitl/rearrange/README.md

```diff
@@ -0,0 +1,20 @@
+# Rearrange HITL application
+
+The user-controlled human and policy-controlled robot must accomplish a collaborative rearrangement task in HSSD scenes. See on-screen help for keyboard/mouse controls. This is similar to the human-in-the-loop tool used in the [Habitat 3.0 paper](https://arxiv.org/abs/2310.13724) to evaluate policies against real human collaborators.
```

Comment:
It would be nice! It's outside the scope of this PR, which is mostly about moving files around. Do you want to suggest this as a task under the [HITL OSS Asana parent task](https://app.asana.com/0/1205829436198608/1206145143211296)?